src/utils/helpers/prices.ts

```diff
@@ -1,32 +1,12 @@
 import { CurrencyIdLiteral } from '@interlay/interbtc-api';
 
-import { Price, Prices } from '@/common/types/util.types';
 import { ForeignAssetIdLiteral } from '@/types/currency';
 
+import { Price, Prices } from '../hooks/api/use-get-prices';
+
 const getTokenPrice = (
   prices: Prices | undefined,
-  tokenIdLiteral: CurrencyIdLiteral | ForeignAssetIdLiteral
-): Price | undefined => {
-  if (!prices) return;
-
-  switch (tokenIdLiteral) {
-    case ForeignAssetIdLiteral.BTC:
-      return prices.bitcoin;
-    case CurrencyIdLiteral.DOT:
-      return prices.polkadot;
-    case CurrencyIdLiteral.INTR:
-      return prices.interlay;
-    case CurrencyIdLiteral.INTERBTC:
-      return prices['interlay-btc'];
-    case CurrencyIdLiteral.KSM:
-      return prices.kusama;
-    case CurrencyIdLiteral.KINT:
-      return prices.kintsugi;
-    case CurrencyIdLiteral.KBTC:
-      return prices['kintsugi-btc'];
-    default:
-      return undefined;
-  }
-};
+  tokenIdLiteral: CurrencyIdLiteral | ForeignAssetIdLiteral | string
```

Comment:
Here, I'm wondering if we really need type union or would be good with just `string` type. The `ForeignAssetIdLiteral` type here is also little confusing (not your contribution I know), because it contains Bitcoin and not the foreign assets as in a sense they exist in our parachain, so I would change at least that.

ALSO I think it would be good to change the parameter name to `ticker` in order to start getting rid of the `CurrencyIdLiteral` type that is not really compatible with foreign assets, but maybe that could be done in another PR - I think we do some unnecessary typecasting when calling this function that would need to be removed and code simiplified.